Apps/Sandcastle/gallery/AtmospherePostProcessing.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no"
+    />
+    <meta
+      name="description"
+      content="Apply an atmosphere light scattering post processing stage."
+    />
+    <meta name="cesium-sandcastle-labels" content="Showcases" />
+    <title>Cesium Demo</title>
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script
+      type="text/javascript"
+      src="../../../Build/CesiumUnminified/Cesium.js"
+      nomodule
+    ></script>
+    <script type="module" src="../load-cesium-es6.js"></script>
+  </head>
+  <body
+    class="sandcastle-loading"
+    data-sandcastle-bucket="bucket-requirejs.html"
+  >
+    <style>
+      @import url(../templates/bucket.css);
+    </style>
+    <div id="cesiumContainer" class="fullSize"></div>
+    <div id="loadingOverlay"><h1>Loading...</h1></div>
+    <div id="toolbar"></div>
+    <script id="cesium_sandcastle_script">
+      function startup(Cesium) {
+        "use strict";
+        //Sandcastle_Begin
+
+        const worldTerrain = Cesium.createWorldTerrain({
+          requestWaterMask: true,
+          requestVertexNormals: true,
+        });
+
+        const viewer = new Cesium.Viewer("cesiumContainer", {
+          terrainProvider: worldTerrain,
+        });
+
+        if (
+          !Cesium.PostProcessStageLibrary.isAtmosphereScatteringSupported(
+            viewer.scene
+          )
+        ) {
+          window.alert(
+            "This browser does not support the Atmosphere Scattering post process."
+          );
+        }
+
+        // required configuration
+        viewer.scene.skyAtmosphere.show = false;
+        viewer.scene.globe.showGroundAtmosphere = false;
+
+        // add post processing stage for astmosphere light scattering
+        viewer.scene.postProcessStages.add(
+          Cesium.PostProcessStageLibrary.createAtmosphereScatteringStage()
+        );
+
+        // set a picturesque time and location
+        viewer.camera.setView({
+          destination: Cesium.Cartesian3.fromDegrees(-0.6, 43.1, 3000.0),
+          orientation: { heading: -1.7, pitch: -0.2, roll: 0.0 },
+        });
+        viewer.clock.currentTime = Cesium.JulianDate.fromDate(
+          new Date("2022-02-02T17:24:00")
```

Comment:
```suggestion
          new Date("2022-02-02T17:24:00+01:00")
```

I added the offset since, depending on your machine's timezone, this may end up showing the dark side of the globe at night.

Reply:
Well spotted, thank you @ggetz